sys/linux/dev_kvm.txt

```diff
@@ -406,6 +423,36 @@ kvm_device_attr {
 	addr	ptr64[in, int64]
 }
 
+kvm_device_attr_vm [
+	attr_arm64	kvm_device_attr_vm_arm64
+	attr_other	kvm_device_attr
+]
+
+kvm_vm_attr_group_arm64 = KVM_ARM_VM_SMCCC_CTRL
+kvm_vm_attr_attr_arm64 = KVM_ARM_VM_SMCCC_FILTER
+
+kvm_device_attr_vm_arm64 {
+	flags	const[0, int32]
+	group	flags[kvm_vm_attr_group_arm64, int32]
+	attr	flags[kvm_vm_attr_attr_arm64, int64]
+	addr	ptr64[in, kvm_device_attr_vm_arg]
+}
+
+kvm_smccc_filter_action = KVM_SMCCC_FILTER_HANDLE, KVM_SMCCC_FILTER_DENY, KVM_SMCCC_FILTER_FWD_TO_USER
+
+kvm_smccc_filter {
+	base		int32
+	nr_functions	int32
+	action		flags[kvm_smccc_filter_action, int8]
+	pad		array[const[0, int8], 15]
+}
+
+# unions need at least 2 fields, but we have only 1 now, but we want to have it as union for future extention
```

Comment:
Why not use something like `type kvm_device_attr_vm_arg kvm_smccc_filter` then? Or just use `kvm_smccc_filter` in `kvm_device_attr_vm_arm64` and add a comment there?

Syzkaller will anyway rediscover the coverage once we refactor these descriptions in the future.

Reply:
It will just discard those calls when from the corpus when it first sees them, but I don't think it's a big problem.

Reply:
Ok, sounds good.